code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -413,11 +416,170 @@
 
 	var/mob/living/carbon/C = M
 	var/obj/item/clothing/mask/facehugger/F = C.is_wearing_item(/obj/item/clothing/mask/facehugger, slot_wear_mask)
+	var/obj/item/clothing/mask/facehugger/F2 = C.is_wearing_item(/obj/item/clothing/mask/facehugger, slot_head)
 
 	if(F && (!F.sterile || hugger.sterile) && F != hugger) // Lamarr won't fight over faces and neither will normal huggers.
 		return FALSE
+	if(F2 && (!F2.sterile || hugger.sterile) && F2 != hugger) 
+		return FALSE
 
 	return TRUE
 
 /obj/item/clothing/mask/facehugger/acidable()
 	return sterile
+
+
+
+
+//////////////////////////////////////////////////// 
+////////////////  HEADCRABS  /////////////////////// 
+////////////////////////////////////////////////////
+
+/obj/item/clothing/mask/facehugger/headcrab
```

Comment:
uhhh, did you add a Process() anywhere to handle headcrab behavior?

Reply:
Why would you need a process? Nothing is changing state dynamically, everything seems to be safely handled when the headcrab is equipped/removed.

Reply:
It inherits process() from facehuggers, but I should move things like the damage while latched on to a process()